### DISCUSSION TITLE
Push 2 via Driven By Moss - not recording notes in Playtime2

### DISCUSSION THREAD

**Discussion by genmce** [helgoboss/helgobox#1314] [Q&A]
I'm using v2.16.13#0 on Windows 10 64 with driven by moss 24.5.1 with push 2 device.
I'm also using a novation xstation for midi input. There are a few odd things happening for me.
I apologize for the length of this 

1. I load a plugin on column 1 by clicking FX -  
*note (my plugin browser is docked so the plugin window never shows up, I have to downsize PlayTime or undock plugin browser - kinda of weird when I watch the video where you make a song - I probably just need to get used to that.)
2. Load a plugin - ok no prob
3. Midi input set to all - I can play notes on the push 2 (driven by moss uses the virtual midi keyboard for input from push2 - if I set midi input to virtual midi keyboard - I hear played notes)
4. Record a clip - I have metronome on, I see count down (I have count-in 2 bars enabled with metronome). Light turns red on clip it is recording, I can hear notes I am playing.
5. Playback previously recorded clip - midi notes were not recorded - or only the first note is recorded (I have not isolated when or why sometimes 1 note or sometimes no midi notes are recorded.)
6. If I use the built in - Reaper - Virtual Midi Keyboard -  to record a clip - notes are recorded.
7. If I use my novation xstation (only as controller - Behringer xr18 is audio device)  - input set on all midi inputs - I don't hear notes when auditioning  and I don't hear notes played when recording, however clip is recorded when I playback the clip, I can hear recorded notes.
8. If I use xstation with midi input set to xstation I can hear notes when armed (auditioning) - I can record and playback as expected.

**Comment by helgoboss** [comment]
Sorry, I haven't looked into DBM support so far, and also not into Push 2. Too many unknowns. I'm afraid this topic needs to wait.

**Reply by genmce** [reply]
New dev release 7.33+dev0213 seems to fix this. Will test again on next official release.  Now I need to learn to use Playtime 2.

**Reply by helgoboss** [reply]
Ah yes. Totally forgot that there was an official issue for this. Justin fixed a related bug in REAPER.
